src/modules/Home/components/Content/index.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import {
+  Container,
+  Grid,
+  Typography,
+  Paper,
+  Divider,
+  TextField,
+  IconButton,
+} from '@material-ui/core/';
+import { SearchOutlined } from '@material-ui/icons';
+import Article from '../Articles';
+import articles from '../../constants';
+import Chips from '../../../shared/Chips';
+
+import './Content.css';
+
+function Content() {
+  const [inputValue, setInputValue] = useState('');
+  const [chipsValue, setChipsValue] = useState('');
```

Comment:
You can combine these states into the one object
`const [appliedFilters, setAppliedFilters] = useState({search: '', filter: ''});`

Reply:
combined the states into one object